fulp_modules/jobs/deputy/code/deputy_zclothing.dm

```diff
@@ -0,0 +1,250 @@
+	desc = "Perhaps the only thing standing between the supermatter and a station-wide explosive sabotage."
+	icon_state = "beret_engi"
+
+/obj/item/clothing/head/fulpberet/medical
+	name = "medical deputy beret"
+	desc = "This proud white-blue beret is a welcome sight when the greytide descends on chemistry."
+	icon_state = "beret_medbay"
+
+/obj/item/clothing/head/fulpberet/science
+	name = "science deputy beret"
+	desc = "This loud purple beret screams 'Dont mess with his matter manipulator!'"
+	icon_state = "beret_science"
+
+//Headsets
+/obj/item/encryptionkey/headset_depsupply
+	name = "supply deputy radio encryption key"
+	icon_state = "sec_cypherkey"
+	channels = list(RADIO_CHANNEL_SECURITY = 1, RADIO_CHANNEL_SUPPLY = 1)
+
+/obj/item/encryptionkey/headset_depengineering
+	name = "engineering deputy radio encryption key"
+	icon_state = "sec_cypherkey"
+	channels = list(RADIO_CHANNEL_SECURITY = 1, RADIO_CHANNEL_ENGINEERING = 1)
+
+/obj/item/encryptionkey/headset_depmedical
+	name = "medical deputy radio encryption key"
+	icon_state = "sec_cypherkey"
+	channels = list(RADIO_CHANNEL_SECURITY = 1, RADIO_CHANNEL_MEDICAL = 1)
+
+/obj/item/encryptionkey/headset_depscience
+	name = "science deputy radio encryption key"
+	icon_state = "sec_cypherkey"
+	channels = list(RADIO_CHANNEL_SECURITY = 1, RADIO_CHANNEL_SCIENCE = 1)
+
+/obj/item/radio/headset/headset_sec/department/supply
+	keyslot = new /obj/item/encryptionkey/headset_depsupply
+
+/obj/item/radio/headset/headset_sec/department/engi
+	keyslot = new /obj/item/encryptionkey/headset_depengineering
+
+/obj/item/radio/headset/headset_sec/department/med
+	keyslot = new /obj/item/encryptionkey/headset_depmedical
+
+/obj/item/radio/headset/headset_sec/department/sci
+	keyslot = new /obj/item/encryptionkey/headset_depscience
+
+//Base Skillchip
+/obj/item/skillchip/job/deputy
+	name = "D3PU7Y skillchip"
+	desc = "You think Deputies learn this stuff naturally?"
+	skill_icon = "sitemap"
+	var/deputy
+	var/department
+	var/datum/martial_art/deputygrab/style
+
+/obj/item/skillchip/job/deputy/Initialize()
+	. = ..()
+	if(deputy)
+		name = "[deputy]-D3PUT7 skillchip"
+	if(department)
+		skill_name = "[department] Deputy"
+		skill_description = "Recognizes [department] as their home, and has a greater fighting advantage while in it."
+		activate_message = "<span class='notice'>You suddenly feel safe in [department].</span>"
+		deactivate_message = "<span class='notice'>[department] no longer feels safe.</span>"
+	style = new
+
+/obj/item/skillchip/job/deputy/on_activate(mob/living/carbon/user, silent = FALSE)
+	. = ..()
+	style.teach(user, make_temporary = TRUE)
+
+/obj/item/skillchip/job/deputy/on_deactivate(mob/living/carbon/user, silent = FALSE)
+	style.remove(user)
+	return ..()
+
+//Supply Skillchip
+/obj/item/skillchip/job/deputy/supply
+	deputy = "5UPP1Y"
+	department = "Cargo"
+	auto_traits = list(TRAIT_SUPPLYDEPUTY, TRAIT_NOGUNS) // Deputies aren't trained to use guns
```

Comment:
i can go to library and remove my skillchip to get access to guns while i only lose a meh mood buff and a meh martial art

Reply:
Then you can go do that, deputies aren’t meant to use guns, the skillchip is to prevent them from just rolling Deputy for the grab and mood boost, then becoming an officer anyways.